chain/chain/src/flat_storage_resharder.rs

```diff
@@ -181,6 +181,22 @@ impl FlatStorageResharder {
         info!(target: "resharding", ?split_params, "initiating flat storage shard split");
```

Comment:
Can you add a comment to the method about the fork handling? 
github doesn't let me comment there but I mean split_shard a few lines above here.

Reply:
Sure